cleaned_data/loganXML/20220114_Braddock_Paige_edited.xml

```diff
@@ -0,0 +1,166 @@
+      xmlns:xs="http://www.w3.org/2001/XMLSchema"
+      xsi:schemaLocation="http://www.loc.gov/mods/v3 http://www.loc.gov/standards/mods/v3/mods-3-5.xsd">
+   <identifier type="local">20220114_Braddock_Paige_edited</identifier>
+   <titleInfo>
+      <title>Interview with Paige Braddock, 2022-01-14</title>
+   </titleInfo>
+   <abstract>Interview with Paige Braddock, Chief Creative Officer for Charles Shulz's studio and former East Tennessee resident, conducted by Casey Kaufman. Braddock discusses her time in East Tennessee and relates her California wildfire experiences to the wildfire in Gatlinburg. She explains how her experiences have helped her create artwork for the Gatlinburg wildfire. Conducted over Zoom due to the COVID-19 pandemic. This is a representative quote from the interview: "If there is a story or an event or something that is almost impossible to illustrate with a photograph, sometimes a drawing or illustration can get to the heart of that so that you're not so much showing a representation of something that happened, but you're getting at the emotion behind what happened."</abstract>
+   <name>
+      <namePart>Braddock, Paige</namePart>
+      <role>
+         <roleTerm authority="marcrelator"
+                   valueURI="http://id.loc.gov/vocabulary/relators/ive">Interviewee</roleTerm>
+      </role>
+   </name>
+   <name>
+      <namePart>Kaufman, Casey</namePart>
+      <role>
+         <roleTerm authority="marcrelator"
+                   valueURI="http://id.loc.gov/vocabulary/relators/ivr">Interviewer</roleTerm>
+      </role>
+   </name>
+   <originInfo>
+      <dateCreated encoding="edtf" keyDate="yes">2022-01-14</dateCreated>
+      <place>
+         <placeTerm valueURI="http://id.loc.gov/authorities/names/n50071679">Murfreesboro (Tenn.)</placeTerm>
+      </place>
+      <place>
+         <placeTerm valueURI="http://id.loc.gov/authorities/names/n82041165">Sebastopol (Calif.)</placeTerm>
+      </place>
+   </originInfo>
+   <physicalDescription>
+      <form authority="aat" valueURI="http://vocab.getty.edu/aat/300136900">motion pictures (visual works)</form>
+      <extent>00:22:21</extent>
+   </physicalDescription>
+   <subject displayLabel="Narrator Class">
+      <topic>Artists</topic>
+   </subject>
+   <subject authority="lcsh"
+      valueURI="http://id.loc.gov/authorities/subjects/sh85146723">
+      <topic>Wildfires</topic>
+   </subject>
+   <subject authority="lcsh"
+      valueURI="http://id.loc.gov/authorities/subjects/sh85008298">
+      <topic>Artists and community</topic>
+   </subject>
+   <subject authority="lcsh"
+      valueURI="http://id.loc.gov/authorities/subjects/sh97000057">
+      <topic>Disasters in art</topic>
+   </subject>
+   <subject authority="geonames" valueURI="http://sws.geonames.org/4624443">
+      <geographic>Gatlinburg</geographic>
+      <cartographics>
+         <coordinates>35.71453, -83.51189</coordinates>
+      </cartographics>
+   </subject>
```

Comment:
I've remembered that for our geographic information to be associated with a Browse category on rfta.lib.utk.edu we need to somewhat duplicate the geographic subject in another field. Can you add the following to all artist records (assuming they focus on Gatlinburg)?

`<note displayLabel="Browse">Gatlinburg, Sevierville and Pittman Center</note>`